fastlane/Fastfile

```diff
@@ -384,4 +394,14 @@ platform :android do
       mapping: "authenticator/build/outputs/mapping/release/mapping.txt",
     )
   end
+
+  desc "Get latest Authenticator published version"
+  lane :getLatestAuthenticatorVersion do |options|
+      version_codes = google_play_track_version_codes(
+        package_name: "com.bitwarden.authenticator",
+        track: "production",
+        json_key: options[:serviceCredentialsFile]
+      )
+      puts version_codes.max
```

Comment:
Just hit me that this should be retrieving the version code (e.g. `20100` in the [last release](https://github.com/bitwarden/android/releases/tag/v2025.4.0)) and the workflow is comparing it to the release tag (e.g. `v2025.4.0`).